GCMessengerSDKSample/app/src/main/java/com/genesys/cloud/messenger/sample/MainActivity.kt

```diff
@@ -328,6 +357,10 @@ class MainActivity : AppCompatActivity(), ChatEventListener {
                     TAG,
                     "!!!!! Chat ${error.scope} can't be created: $message"
                 )
+
+                findChatFragment()?.isVisible?.let {
```

Comment:
I need to verify that the chat fragment is visible when the chat loading error occurs in order to return to the chat form screen. I was able to reproduce this scenario only in activity recreation flow (Don't keep activities).
In the regular chat creation flow we don't show the chat fragment in case of error.

Reply:
i was referring to the syntax. 
`findChatFragment()?.isVisible?.let` - if `isVisible` == false will it go into the `let` block